msdk-id-sirius/src/main/java/SiriusIdentificationMethod.java

```diff
@@ -0,0 +1,95 @@
+import de.unijena.bioinf.ChemistryBase.chem.PrecursorIonType;
+import de.unijena.bioinf.ChemistryBase.ms.Ms2Experiment;
+import de.unijena.bioinf.ChemistryBase.ms.Peak;
+import de.unijena.bioinf.ChemistryBase.ms.Spectrum;
+import de.unijena.bioinf.sirius.IdentificationResult;
+import de.unijena.bioinf.sirius.IsotopePatternHandling;
+import de.unijena.bioinf.sirius.Sirius;
+import io.github.msdk.MSDKException;
+import io.github.msdk.MSDKMethod;
+import io.github.msdk.MSDKRuntimeException;
+import io.github.msdk.datamodel.IonAnnotation;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import javax.annotation.Nullable;
+
+/**
+ * Created by evger on 14-May-18.
+ */
+
+public class SiriusIdentificationMethod implements MSDKMethod<IonAnnotation> {
+  Sirius sirius;
```

Comment:
if this variable never changes, make it final. always use final when possible: http://www.javapractices.com/topic/TopicAction.do?Id=23